code/modules/power/apc.dm

```diff
@@ -803,12 +803,15 @@
 	var/datum/species/ethereal/eth_species = H.dna?.species
 	if(istype(H) && istype(eth_species))
 		if(H.a_intent == INTENT_HARM)
-			if(eth_species.ethereal_charge >= ETHEREAL_CHARGE_FULL)
+			if(eth_species.ethereal_charge >= ETHEREAL_CHARGE_FULL - 19)
 				to_chat(H, "<span class='warning'>Your charge is full!</span>")
 				return
+					if(cell.charge < 100)
```

Comment:
I told you to ignore discord formatting, this code is indented too far and should be aligned with the previous check.